lib/core/numbers.toit

```diff
@@ -954,6 +954,17 @@ abstract class int extends num:
   stringify radix/int -> string:
     #primitive.core.int64-to-string
 
+  /**
+  Variant of $(stringify).
+
+  Treats the number as an unsigned 64-bit integer.
+  */
+  stringify --unsigned-64/True -> string:
```

Comment:
Don't we use the int64 abbreviation in other places? I think it would be logical if we called this `stringify --uint64`.

Reply:
I initially only had `stringify --unsigned`, and then decided to mention the 64.
Going with your suggestion now.
done.